services/sqs/src/main/resources/codegen-resources/customization.config

```diff
@@ -12,6 +12,7 @@
         }
     },
 
-    "enableGenerateCompiledEndpointRules": true
+    "enableGenerateCompiledEndpointRules": true,
+    "batchManagerSupported": true
```

Comment:
Should this be more generic now that batch managers could differ between services, e.g. where the actual FQCN, construct method, and argument are all specified? Maybe similar to the way the `utilities()` method is generated? 

https://github.com/aws/aws-sdk-java-v2/blob/4f967b987d651a8438fff99604127f19df676218/services/s3/src/main/resources/codegen-resources/customization.config#L285